### DISCUSSION TITLE
is there a way that I can protect & hide the CLI commands ?

### DISCUSSION THREAD

**Discussion by Omar7saeed** [Tyrrrz/CliWrap#122] [Q&A] [answered]
hello  : )

I have 2 question 

I want to hide all process of CLI in the source code 
is there a way that I can apply Using cryptography in .net

2- in your opinion bro .. what is the best way to protect source code from reverse engineering at this time ?

sorry for asking many times ... but it's the first time that I find someone who replies and helps & support me in this field 
I'm just **15** years old by the way.

@Tyrrrz  🥇

**Comment by Tyrrrz** [comment, marked as answer]
Hello

1. No, it's not possible.
2. Your best option is probably various .NET obfuscators, but ultimately it will be a game of cat and mouse
